src/test/java/com/opentok/test/OpenTokTest.java

```diff
@@ -237,7 +237,7 @@ public void testTokenDefault() throws
 
         int apiKey = 123456;
         String apiSecret = "1234567890abcdef1234567890abcdef1234567890";
-        OpenTok opentok = new OpenTok(apiKey, apiSecret);
+        OpenTok opentok = new OpenTok.Builder(apiKey, apiSecret).build();
```

Comment:
Once you make the one constructor with just the mandatory parameters public, you can stop using the Builder in all the test cases except those which use the optional properties.